src/app/sign-up/page.tsx

```diff
@@ -1,48 +1,86 @@
 "use client";
 
-import { SetStateAction, useState } from "react";
-import clsx from "clsx"; // Import clsx
+import { useState } from "react";
+import clsx from "clsx";
 import styles from "./page.module.css";
 import { Newsreader } from "@next/font/google";
+import axios from "axios";
 
 const newsreader = Newsreader({
   weight: "700",
   subsets: ["latin"],
 });
 
+interface User {
+  names: string;
+  email: string;
+  password: string;
+  role: string;
+}
+
+export const createPost = async (postData: User) => {
+  console.log(postData);
+  try {
+    const response = await axios.post(
+      "https://express-sequelize-api-xgit.onrender.com/user/signup",
```

Comment:
Store this URL as an environment variable in .env file, read more [https://www.thirdrocktechkno.com/blog/how-to-set-environment-variable-in-node-js/](https://www.thirdrocktechkno.com/blog/how-to-set-environment-variable-in-node-js/)